scripts/bolt-instrumented.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# This file captures profile data for bolt
+#
+# The profile data is captured by binary instrumentation via bolt.
+# It therefore does not require a CPU with a last branch register and can run on both AMD and INTEL
+
+CMAKE_DIR="cmake-build-bolt-instrumented"
+
+mkdir "$CMAKE_DIR"
+
+pushd "$CMAKE_DIR"
+cmake -DCMAKE_UNITY_BUILD=ON -GNinja -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCOMPILE_FOR_BOLT=TRUE ..
+ninja all -j $(nproc)
+
+mv lib/libhyrise_impl.so lib/libhyrise_impl.so.source
+llvm-bolt lib/libhyrise_impl.so.source -instrument -o lib/libhyrise_impl.so
+popd
+
+for benchmark in hyriseBenchmarkTPCH hyriseBenchmarkTPCDS hyriseBenchmarkTPCC hyriseBenchmarkJoinOrder hyriseBenchmarkStarSchema
+do
+    # TODO: Multi-threaded benchmarks?
```

Comment:
Yes, please. And please let it run for at least 20mins. The first minute is only loading data for some benchmarks. And probably also use a couple of clients. At least 8 or so ...